package.json

```diff
@@ -55,14 +61,15 @@
     "cpx": "^1.5.0",
     "eslint": "^5.15.2",
     "jest": "^24.5.0",
+    "ng-packagr": "^5.7.1",
     "rimraf": "^2.6.2",
-    "single-spa": "^4.3.5",
+    "single-spa": "^4.4.1",
     "ts-jest": "^24.0.0",
     "typescript": "3.2.4",
     "webpack": "^4.29.6"
   },
   "dependencies": {
     "semver": "^6.2.0",
-    "single-spa": ">= 4.0.0"
+    "single-spa": "^4.4.1"
```

Comment:
I prefer not changing this -- single-spa-angular really does work with anything `=> 4.0.0`. We are going to release single-spa@5 in the next few weeks and it will work with single-spa-angular. The change you have here would prevent people from using single-spa@5, whereas the `>=` allows them to use single-spa@5

Reply:
Was my mistake, but fixed